### PR TITLE
Content layout created

### PR DESCRIPTION
**Pull request by nvescio**
Summary
 - Content layout

Screenshots
Chrome
contentlayout

Trello Card
https://trello.com/c/mMQAobZb/23-content-layout

### CODE REVIEW

File: training_maquetado/index.html [hunk 1]
```diff
@@ -16,12 +16,83 @@
...
-  <div class="tech-container"> 
-        <img class="tech-icon" src="assets/react_icon.svg">
-        <h2 class="tech-title">ReactJS</h2>
-        <h2 class="tech-release">Release: 2013</h2>
-        <p class="tech-desc">React is a declarative, component-based JavaScript library for building user interfaces.</p>
-  </div>   
+  <div class="tech-grid">
+
+    <div class="tech-container"> 
+          <img class="tech-icon" src="assets/react_icon.svg">
```
Comment (CarlosHWolox) [inline training_maquetado/index.html]: indentation

File: training_maquetado/index.html [hunk 2]
```diff
@@ -16,12 +16,83 @@
...
+
+  <div class="title-container"> 
+    <img class="triangle-decorator" src="assets/triangle.svg">
+    <h1 class="title">Back-End Tools</h1>
+  </div> 
+
+  <div class="tech-grid">
+
+    <div class="tech-container"> 
+          <img class="tech-icon" src="assets/rails_icon.png">
```
Comment (CarlosHWolox) [inline training_maquetado/index.html]: indentation

File: training_maquetado/scss/tech.scss [hunk 3]
```diff
@@ -20,28 +21,42 @@
     
     &,
     &::before {
-      border-radius: 10px;
+      border-radius: 6px;
       box-sizing: border-box;
+      margin: 2% 0;
```
Comment (CarlosHWolox) [inline training_maquetado/scss/tech.scss]: Try not to use percent and use px values

File: training_maquetado/scss/tech.scss [hunk 4]
```diff
@@ -3,9 +3,10 @@
   background-color: $green;
   display: flex;
   flex-direction: column;
-  height: 300px;
+  height: 350px;
```
Comment (CarlosHWolox) [inline training_maquetado/scss/tech.scss:16]: you can define a min-height value

File: training_maquetado/scss/tech.scss [hunk 5]
```diff
@@ -3,9 +3,10 @@
...
+  padding: 12px 12px;
```
Comment (CarlosHWolox) [inline training_maquetado/scss/tech.scss]: you can center the content with align-content so you dont need this paddings

File: training_maquetado/scss/tech.scss [hunk 6]
```diff
@@ -3,9 +3,10 @@
...
   position: relative;
```
Comment (CarlosHWolox) [inline training_maquetado/scss/tech.scss:17]: i think this is not necessary

File: training_maquetado/scss/tech.scss [hunk 7]
```diff
@@ -20,28 +21,42 @@
...
 }
+
+.tech-grid {
+  align-content: space-between;
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  justify-content: space-between;
+  margin: 0 auto 80px;
+  width: 55%;
```
Comment (CarlosHWolox) [inline training_maquetado/scss/tech.scss]: Using percent is not necessary, with just margin, padding and width you can center the elements

File: training_maquetado/scss/titles.scss [hunk 8]
```diff
@@ -8,6 +8,11 @@
   font-size: $text-big;
   height: $title-decoration-height;
    
+    &,
```
Comment (CarlosHWolox) [inline training_maquetado/scss/titles.scss]: &, &::before?

File: training_maquetado/scss/titles.scss [hunk 9]
```diff
@@ -8,6 +8,11 @@
...
+    &::before {
+        padding: 5% 20%;
```
Comment (CarlosHWolox) [inline training_maquetado/scss/titles.scss]: percent is not necessary, use top or margin on the title root class

File: training_maquetado/scss/tech.scss [hunk 10]
```diff
@@ -1,15 +1,26 @@
...
 .tech-container {
   align-items: center;
   background-color: $green;
+  border-radius: 7px; 
   display: flex;
   flex-direction: column;
-  height: 350px;
-  padding: 12px 12px;
+  justify-content: center;
+//   padding: 12px 12px;
```
Comment (CarlosHWolox) [inline training_maquetado/scss/tech.scss]: Delete comment

File: training_maquetado/index.html [hunk 11]
```diff
@@ -11,11 +11,66 @@
...
   <h1 class="title">Front-End Tools</h1>
-  <div class="tech-container"> 
-    <img class="tech-icon" alt="ReactJS icon" src="assets/react_icon.svg">
-    <h2 class="tech-title">ReactJS</h2>
-    <h4 class="tech-release">Release: 2013</h4>
-    <p class="tech-desc">React is a declarative, component-based JavaScript library for building user interfaces.</p>
-  </div>
+  <div class="tech-grid">
+    <div class="tech-container"> 
+      <img class="tech-icon" alt="ReactJS icon" src="assets/react_icon.svg">
```
Comment (FrankIglesias) [inline training_maquetado/index.html]: add a slash before the >

File: training_maquetado/index.html [hunk 12]
```diff
@@ -11,11 +11,66 @@
...
+      <h2 class="tech-title">ReactJS</h2>
+      <h4 class="tech-release">Release: 2013</h4>
+      <p class="tech-desc">React is a declarative, component-based JavaScript library for building user interfaces.</p>
+    </div>
+    <div class="tech-container"> 
+      <img class="tech-icon" src="assets/angular_icon.svg">
```
Comment (FrankIglesias) [inline training_maquetado/index.html]: add a slash before the > and an alt

File: training_maquetado/index.html [hunk 13]
```diff
@@ -11,11 +11,66 @@
...
+      <h2 class="tech-title">AngularJS</h2>
+      <h4 class="tech-release">Release: 2010</h4>
+      <p class="tech-desc">AngularJS (commonly reffered to as AngularJS 1.X) is a JavaScript framework for building SPAs with an MVC architecture.</p>
+    </div>    
+    <div class="tech-container"> 
+      <img class="tech-icon" src="assets/vue_icon.svg">
```
Comment (FrankIglesias) [inline training_maquetado/index.html]: add a slash before the > and an alt

File: training_maquetado/index.html [hunk 14]
```diff
@@ -11,11 +11,66 @@
...
+      <h2 class="tech-title">VueJS</h2>
+      <h4 class="tech-release">Release: 2014</h4>
+      <p class="tech-desc">Vue.js (pronounced /vjuː/, like view) is an open-source JavaScript framework for building user interfaces.</p>
+    </div>     
+    <div class="tech-container"> 
+      <img class="tech-icon" src="assets/elm_icon.svg">
```
Comment (FrankIglesias) [inline training_maquetado/index.html]: add a slash before the > and an alt

File: training_maquetado/index.html [hunk 15]
```diff
@@ -11,11 +11,66 @@
...
+      <img class="tech-icon" src="assets/polymer_icon.svg">
+      <h2 class="tech-title">Polymer</h2>
+      <h4 class="tech-release">Release: 2015</h4>
+      <p class="tech-desc">Polymer is an open-source JavaScript library for building web applications using Web Components.</p>
+    </div>  
+  </div>  
+  <h1 class="title">Back-End Tools</h1>
+  <div class="tech-grid">
+    <div class="tech-container"> 
+      <img class="tech-icon" src="assets/rails_icon.png">
```
Comment (FrankIglesias) [inline training_maquetado/index.html]: add a slash before the > and an alt

File: training_maquetado/index.html [hunk 16]
```diff
@@ -11,11 +11,66 @@
...
+      <h2 class="tech-title">Ruby on Rails</h2>
+      <h4 class="tech-release">Release: 2005</h4>
+      <p class="tech-desc">Ruby on Rails, or Rails, is a server-side web application framework written in Ruby. Rails is an MVC framework, providing default structures for a database, a web service, and web pages.</p>
+    </div>
+    <div class="tech-container"> 
+      <img class="tech-icon" src="assets/express_icon.svg">
```
Comment (FrankIglesias) [inline training_maquetado/index.html]: add a slash before the > and an alt

File: training_maquetado/index.html [hunk 17]
```diff
@@ -11,11 +11,66 @@
...
+      <h2 class="tech-title">NodeJS</h2>
+      <h4 class="tech-release">Release: 2009</h4>
+      <p class="tech-desc">Node.js is a JavaScript runtime built on Chrome's V8 JavaScript engine. Uses an event-driven, non-blocking I/O model taht makes it lightweight and efficient.
+      </p>
+    </div>    
+    <div class="tech-container"> 
+      <img class="tech-icon" src="assets/aspnet_icon.png">
```
Comment (FrankIglesias) [inline training_maquetado/index.html]: add a slash before the > and an alt

File: training_maquetado/index.html [hunk 18]
```diff
@@ -11,11 +11,66 @@
...
+      <h2 class="tech-title">ASP.NET Core</h2>
+      <h4 class="tech-release">Release: 2016</h4>
+      <p class="tech-desc">ASP.NET Core is the next generation of ASP.NET. It is a modular framework that runs on both the full .NET Framework, on Windows, and the cross-platform .NET Core.
+      </p>
+    </div>     
+    <div class="tech-container"> 
+      <img class="tech-icon" src="assets/django_icon.svg">
```
Comment (FrankIglesias) [inline training_maquetado/index.html]: add a slash before the > and an alt

File: training_maquetado/scss/tech.scss [hunk 19]
```diff
@@ -1,14 +1,26 @@
...
+
 .tech-container {
   align-items: center;
   background-color: $green;
+  border-radius: 7px; 
   display: flex;
   flex-direction: column;
-  height: 300px;
+  justify-content: center;
+//   padding: 12px 12px;
```
Comment (FrankIglesias) [inline training_maquetado/scss/tech.scss]: remove commented line

File: training_maquetado/scss/tech.scss [hunk 20]
```diff
@@ -1,14 +1,26 @@
...
+  margin: 30px 0;
+  min-height: 350px;
```
Comment (FrankIglesias) [inline training_maquetado/scss/tech.scss]: I think this is a height value

**Review by FrankIglesias — CHANGES_REQUESTED** [review]

**Review by CarlosHWolox — APPROVED** [review]

File: training_maquetado/scss/titles.scss [hunk 21]
```diff
@@ -7,10 +7,12 @@
   display: flex;
   font-size: $text-big;
   height: $title-decoration-height;
+  margin: 0 325px;
```
Comment (FrankIglesias) [inline training_maquetado/scss/titles.scss]: This can be raplaced by a flex prop

**Review by FrankIglesias — CHANGES_REQUESTED** [review]

**Review by FrankIglesias — APPROVED** [review]

**Pull request merged**